pkg/images/images.go

```diff
@@ -25,14 +25,21 @@ import (
 
 	apis "github.com/submariner-io/submariner-operator/api/v1alpha1"
 	v1 "k8s.io/api/core/v1"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+var log = logf.Log.WithName("images")
+
 func GetImagePath(repo, version, image, component string, imageOverrides map[string]string) string {
+	log.Info("GetImage called", "repo", repo, "version", version, "image", image, "component", component)
```

Comment:
```suggestion
	log.Info("GetImagePath called", "repo", repo, "version", version, "image", image, "component", component)
```

The exact function name is more clear and easier to reference the log line with the function name.